ci-operator/step-registry/openshift/e2e/vsphere/csi/openshift-e2e-vsphere-csi-workflow.yaml

```diff
@@ -1,18 +1,19 @@
 workflow:
   as: openshift-e2e-vsphere-csi
   steps:
+    allow_best_effort_post_steps: true
     pre:
     - chain: ipi-vsphere-pre
     - ref: storage-conf-csi-vsphere
     - ref: storage-conf-feature-gate
     - ref: storage-conf-wait-for-csi-driver
     # Wait for CSI migration enabed by TechPreviewNoUpgrade to restart all nodes
     - ref: storage-conf-wait-for-csi-migration
-    - ref: storage-pv-save
+    - ref: storage-obj-save
     test:
     - ref: openshift-e2e-test
     post:
-    - ref: storage-pv-check
+    - ref: storage-obj-check
```

Comment:
can we stop this from running if starting the cluster failed? It seems pointless to check for storage objects if cluster failed to come up. It causes some noise in CI.

Reply:
This PR makes the `post` steps optional (i.e. do not complain if it fails). It's not possible to skip `pre`  steps though.

I updated `storage-obj-save` to never fail, see the script.